### ISSUE TITLE
Quark Branding V1

### ISSUE THREAD

**Issue by ccioanca** [jscherer92/Quark#17]
I know @jscherer92 said to hold off on the branding for Quark. But I had a little spare time on my hands, and a spark for an idea, so I decided to make a thing. It's gotten positives from the few people I've showed it to, so I made a presentation format for it, and here it is. 

This might also come in handy if we decide that we should put out some feelers on social media platforms to see if we can recruit some help. People respond better to something visual than just an idea. 

If the logo looks good and we want to go ahead with it, I can send @jscherer92 whatever assets he might want.

![quark pres](https://user-images.githubusercontent.com/13683999/40316061-196d4570-5ceb-11e8-907d-8c4b18875eda.jpg)

[Quark Pres.pdf](https://github.com/jscherer92/Quark/files/2023147/Quark.Pres.pdf)

**Comment by jscherer92** [comment]
@ccioanca I like it! It looks really clean and simple.

I have been having trouble finding time to work on the project but I want to get back to it soon. I may go through and document what I have so people with C++11/14 knowledge can help with this.

**Comment by ccioanca** [comment]
I havent done any C++ in years, and I havent done any advanced C++ ever. But I'll try to make heads or tails from your documentation, and see what I can contribute. I'll also try to spread the word.

**Comment by kormyen** [comment]
Awesome logo @jscherer92. I don't know about the orange 'k' or font but I love the symbol.

**Comment by jscherer92** [comment]
All credit goes to @cciocanca! Sorry I have not been active as of late! I'm diving deep into Microsoft's COM system and trying to make sure I use it correctly while also working on a personal project using Kotlin.

**Comment by ccioanca** [comment]
@kormyen yeah, i think i agree with you. I'd also probably change the typeface, and maybe play around with more color options. I'll do more work on it when i have any ideas
